test/integration/device-api/v1.spec.ts

```diff
@@ -38,4 +38,17 @@ describe('device-api/v1', () => {
 			await request(api).get('/v1/healthy').expect(500);
 		});
 	});
+
+	describe('POST /v1/blink', () => {
+		// Actions are tested elsewhere so we can stub the dependency here
+		before(() => stub(actions, 'identify'));
+		after(() => (actions.identify as SinonStub).restore());
```

Comment:
Again the fact that the API calls `identify` under the hood is an implementation detail, which tests should not rely on. The right way would probably be to do it passing an identification function to the API object constructor or something like that. For now this is OK since this is a major improvement on what we had before and I don't want to derail the PR. 

Just making sure to leave the comment and this will apply for similar stubs, even if I don't add a comment.